experimental/SymmetricIntersections/README.md

```diff
@@ -0,0 +1,105 @@
+available, the extent of the content as well as the existence of this project
+may vary in the coming months.
+
+Moreover, the current implementation of representations theory of finite groups
+available throughout *should not be taken as a standard* for Oscar: it is a
+temporary solution for the purpose of this project. This part of the code was
+not written by an expert on representation theory, and some of the expensive
+functions might not be computationally optimized. This is also why the content
+is limited to "what was needed". Nevertheless, it was made to be usable by any
+user.
+
+## Content
+
+Here is a list of what kind of functionalities are available:
+
+### Linear representations of finite groups
+
+Basics manipulations of linear representations of finite groups over fields of
+characteristic zero, as well as some complements to character theory, are
+available. Namely, one should be able to create and manipulate linear
+representations of a given finite group over a big enough field *of
+characteristic zero*.
+
+For instance, one can compute the direct sum of representations, the dual
+representation, the symmetric/exterior powers of a representation, but also
+bases for the isotypical components. Finally, few things have been added to
+characters, namely the possibility to switch between characters and
+representations, and enumerating constituents of a given degree.
+
+### Projective representations
+
+These representations are implemented to allow the user to classify, up to
+similarity, faithful projective representations of a finite group over a
+field of characteristic zero. Such representations represent faithful
+linear actions of groups on projective spaces.
+
+The main function from this part is called `faithful_projective_representations`,
+which returns representatives of similarity classes of faithful projective
+representations of a given group, of a given dimension.
+
+*Almost* all the functionalities available for linear representations are
+available for projective representations.
+
+### Symmetric Grassmannians
+
+We call here *Symmetric Grassmannians* spaces which parametrize submodules of a
+given group algebra modules. In some sense, we add a group action to the usual
+Grassmannian variety, and look for invariant subspaces of a fixed vector space
+on which a group acts.
+Even though some functionalities about them are available, the naming
+"Symmetric Grassmannians" is not standard and the usage is quite limited. This
+is why the content related to them is not directly exported to the user:
+one may used "Oscar.bla(arguments, of, bla)" to call the corresponding function
+"bla".
+
+Given a group algebra modules $M$, represented by a linear representation of a
+finite group, one can create the space parametrising submodules $M$ of a given
+dimension, with given character or with given determinant character. Once
+created, it is possible to ask for a *general element*, some parametrization
```

Comment:
general in which sense?

Reply:
Hum I did not chose the best word here, since it is not general.. I will modify this.